Source/WebCore/bindings/js/SerializedScriptValue.cpp

```diff
@@ -5465,25 +5474,25 @@ JSValue SerializedScriptValue::deserialize(JSGlobalObject& lexicalGlobalObject,
 
 JSValue SerializedScriptValue::deserialize(JSGlobalObject& lexicalGlobalObject, JSGlobalObject* globalObject, const Vector<RefPtr<MessagePort>>& messagePorts, const Vector<String>& blobURLs, const Vector<String>& blobFilePaths, SerializationErrorMode throwExceptions, bool* didFail)
 {
-    DeserializationResult result = CloneDeserializer::deserialize(&lexicalGlobalObject, globalObject, messagePorts, WTFMove(m_backingStores)
+    DeserializationResult result = CloneDeserializer::deserialize(&lexicalGlobalObject, globalObject, messagePorts, WTFMove(m_internals.backingStores)
```

Comment:
Can we just pass WTFMove(m_internals) to CloneDeserializer::deserialize instead of all these parameter pieces of it?

Reply:
Not easily. I'll try looking in a follow-up as trying to make this change ended up too large.